scripts/tag.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
```

Comment:
this script from pandas-release.

if we want to trigger actions probably better to have in same repo.